### ISSUE TITLE
Snippets not linked

### ISSUE THREAD

**Issue by rafhun** [jefkoslowski/gitignore-snippets#6]
Installing the package (version 0.2.5) on a fresh install of Atom (version 1.0.2) goes through successfully but the snippet prefixes are not set. So when entering `gig-` no snippet autocomplete shows up:
![settings_-__www_sites_chartreuse_-_atom](https://cloud.githubusercontent.com/assets/7340779/8801433/d99e6222-2fba-11e5-82c1-d90b4883e3fc.png)

I had some snippets defined before installing the package, might these be cause of the problem?

**Comment by jefkoslowski** [comment]
Hi, I could not reproduce the problem. Indeed, the package description looks odd here too, with all those blank lines. But snippets are working fine, using Atom 1.0.2 and gitignore-snippets 0.2.5. Other snippets should not be a problem.

If you can give me steps to reproduce the problem or maybe a video. Thanks.

**Comment by rafhun** [comment]
Hi, I was able to possibly find the problem. Next to this package I have also installed the language-gitignore package to have syntax highlighting for my `.gitignore` files. If I deactivate that or try to use the snippets on files with `Plain Text` they work just fine.

I guess the language file adds a namespace to the buffer (`.source.gitignore`). Might it be possible to extend the snippets for this?
![settings_-__www_sites_styles-library_-_atom](https://cloud.githubusercontent.com/assets/7340779/8821214/5245d592-305c-11e5-9ab4-10f8c366432f.png)

Unfortunately I do not know enough about Atom's snippets yet to determine if the `*` scope you use can easily be extended or why it does not work for the `source.gitignore` scope.
